perfzero/docker/Dockerfile_ubuntu_1804_tf_custom_pip

```diff
@@ -0,0 +1,77 @@
+# Ubuntu 18.04 Python3.6 with CUDA 10 and the following:
+#  - Installs custom TensorFlow pip package
+#  - Installs requirements.txt for tensorflow/models
+
+FROM nvidia/cuda:10.0-base-ubuntu18.04 as base
+ARG tensorflow_pip_spec="resources/tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl"
+
+COPY ${tensorflow_pip_spec} /tensorflow-1.13.1-cp36-cp36m-linux_x86_64.whl
```

Comment:
Let's change the name to  tensorflow-0.0.1-cp36-cp36m-linux_x86_64.whl and add a comment about this being a place holder and all that matters is the cp36-cp36m as pip will not install if that "stuff" is wrong.